benchmark/bench_moe_topk_softmax.py

```diff
@@ -23,6 +24,22 @@ def vllm_topk_softmax(gating_output, topk):
     return topk_weights, topk_indices
 
 
+def navtive_topk_softmax(gating_output, topk):
```

Comment:
This native function does not support `renormalize`. You may follow this one in SGLang:

https://github.com/sgl-project/sglang/blob/36942660513213f3ecd4a39ad64d6cf3127328a9/python/sglang/srt/layers/moe/topk.py#L411-L417